src/compas_timber/ghpython/components/CT_Joint_Rule_Direct/code.py

```diff
@@ -18,7 +18,8 @@ def __init__(self):
         super(DirectJointRule, self).__init__()
         self.classes = {}
         for cls in get_leaf_subclasses(Joint):
-            self.classes[cls.__name__] = cls
+            if cls.MAX_ELEMENT_COUNT == 2:
+                self.classes[cls.__name__] = cls
```

Comment:
so that we get the joints with 2 beams so that we still get 2 named input parameters in GH. The other component takes a list of beams that all get joined in one joint. We could have a differnet attribute like `joint.uses_named_beams` which could be expanded to joints with 3 or more named beam arguments.